index.html

```diff
@@ -1,162 +1,127 @@
       <div class="wrapper">
-        <img src="./assets/img/shape-set-01.png" class="shape-set" alt="" />
-        <h1 id="text">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
-        <h2>"Hello! Sharing world"</h2>
-        <p class="description">
-          You are invited to the thrash Metal Music Conference, Man presenters the metallica 
-          band.</p>
-        </p>
-        <h3>28 December, 2022 </h3>
-        <p class="location">Guayaquil, Ecuador.</p>
+        <a href="#"><i class="fa fa-mastodon"></i></a>
+        <a href="#"><i class="fa fa-friendica"></i></a>
+        <a href="#"><i class="fa fa-twitter"></i></a>
+        <a href="#">Login/Register</a>
       </div>
-    </section>
+    </div>
+    <div id="nav-bar">
+      <a href="https://www.linkedin.com/in/david-verde-3349b114b/e"> <img class="logo"
+          src="./assets/img/logo-david-nuevo-2021.png" alt="logo"> </a>
+      <nav id="menu">
+        <span id="menu-close-btn">x</span>
+        <a href="#" class="home-link">Home</a>
+        <a href="#program">Program</a>
+        <a href="#speakers">Speakers</a>
+        <a href="#sponsors">Sponsors</a>
+        <a href="./about.html">About</a>
+      </nav>
+      <a href="#" id="menu-toggler">
+        <svg class="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+          <path stroke-width="3" d="M4 6h16M4 12h16M4 18h16"></path>
+        </svg>
+      </a>
+    </div>
+  </header>
 
-    <section id="program">
-      <h2 class="section-heading">Main Program</h2>
-      <div class="items">
-        <div class="item">
-          <img src="./assets/img/icons-1.svg" class="item-icon" alt="" />
-          <h3 class="item-title">Lecture</h3>
-          <p class="item-description">
-            Listen to the speakers Experts on Thrash Metal Subject.
-          </p>
-        </div>
-        <div class="item">
-          <img src="./assets/img/icons-2.svg" class="item-icon" alt="" />
-          <h3 class="item-title">Workshop</h3>
-          <p class="item-description">
-            Try creating your music using instruments and experiences in the event.
-          </p>
-        </div>
-        <div class="item">
-          <img src="./assets/img/icons-3.svg" class="item-icon" alt="" />
-          <h3 class="item-title">Forum</h3>
-          <p class="item-description">
-           Share your experience and make some questions to the band.
-          </p>
-        </div>
+  <section id="hero" class="container">
+    <div class="wrapper">
+      <img src="./assets/img/shape-set-01.png" class="shape-set" alt="" />
+      <h1 id="text">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <h2>"Hello! Sharing world"</h2>
+      <p class="description">
+        You are invited to the thrash Metal Music Conference, Man presenters the metallica
+        band.</p>
+      <h3>28 December, 2022 </h3>
+      <p class="location">Guayaquil, Ecuador.</p>
+    </div>
+  </section>
+
```

Comment:
The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)